### PR TITLE
dm-syncer: minor fix on password && print parse error

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 

1. don't decrypt password, will connect to database failed
2.  log is not initial, need to use `fmt` to print parse error

### What is changed and how it works?

fix these two problems

### Check List 

Tests 

 - Manual test

### CODE REVIEW

File: cmd/dm-syncer/main.go [hunk 1]
```diff
@@ -36,13 +36,15 @@ func main() {
...
-	switch errors.Cause(err) {
-	case nil:
-	case flag.ErrHelp:
-		os.Exit(0)
-	default:
-		log.L().Error("parse cmd flags err " + err.Error())
-		os.Exit(2)
+	if err != nil {
+		switch errors.Cause(err) {
+		case nil:
```
Comment (kennytm) [inline cmd/dm-syncer/main.go]: No need to keep the `case nil` here since `err != nil`... Why add the `if err != nil` anyway? `errors.Cause(nil) == nil`.

Comment (WangXiangUSTC) [reply cmd/dm-syncer/main.go]: you are right, update in 6800d8b

File: cmd/dm-syncer/main.go [hunk 2]
```diff
@@ -63,7 +63,13 @@ func main() {
 		log.L().Info("", zap.Stringer("dm-syncer conf", conf))
 	})
 
-	sync := syncer.NewSyncer(conf)
+	conf2, err := conf.DecryptPassword()
```
Comment (csuzhangxc) [inline cmd/dm-syncer/main.go]: Can we let the user use an unencrypted password just as old _Syncer_ did?

Comment (WangXiangUSTC) [reply cmd/dm-syncer/main.go]: update in 820012b

**Pull request merged**